CODEOWNERS

```diff
@@ -208,6 +208,7 @@
 /drivers/i2c/i2c_emul.c                   @sjg20
 /drivers/i2c/i2c_shell.c                  @nashif
 /drivers/i2c/Kconfig.i2c_emul             @sjg20
+/drivers/i2c/slave/                       @henrikbrixandersen
```

Comment:
This may become more than I signed up for in the future 😉 Can we limit it to just this one driver file?

Reply:
There's nothing else in there, but yeah, I can limit it to whatever the check complains about.